src/main/resources/com/google/api/codegen/py/main.snip

```diff
@@ -235,12 +235,17 @@
                         @end
                     @end
 
+                    if metadata is None:
```

Comment:
This is only needed when appending the routing headers. Move it to the routing header conditional.

Reply:
done.